src/com/ms/silverking/cloud/dht/daemon/storage/NamespaceStore.java

```diff
@@ -757,7 +757,7 @@ private long newestVersion(DHTKey key) {
                     ByteBuffer result;
 
                     if (debugWaitFor) {
-                        System.out.printf("pendingWaitFor %s options %s\n", pendingWaitFor, pendingWaitFor.getOptions());
+                        Log.fineAsyncf("pendingWaitFor %s options %s", pendingWaitFor, pendingWaitFor.getOptions());
```

Comment:
debugWaitFor is off by default. Make Log.warningAsyncf so that the debugWaitFor flag has an effect